internal/cli/enable.go

```diff
@@ -14,9 +16,13 @@ var CmdEnable = &cobra.Command{
 	Use:           "enable",
 	Short:         "enable Epinio features",
 	Long:          `enable Epinio features that are not enabled by default`,
-	Args:          cobra.ExactArgs(0),
 	SilenceErrors: true,
 	SilenceUsage:  true,
+	Args:          cobra.MinimumNArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		cmd.Usage()
+		return errors.New(fmt.Sprintf(`Unknown method "%s"`, args[0]))
```

Comment:
nit: there is also https://pkg.go.dev/fmt#Errorf and https://pkg.go.dev/github.com/pkg/errors?utm_source=godoc#Errorf

Reply:
Fixed with commit d96c92939a49edd006e3e7119f6d4157e96c9025